### PR TITLE
qv2ray: new, 2.6.3+git20210102

### PR DESCRIPTION
**Pull request by PegionFish**
Topic Description
-----------------

Package(s) Affected
-------------------

qv2ray

Security Update?
----------------

No

Architectural Progress
----------------------

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-network/qv2ray/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,5 @@
+PKGNAME=Qv2ray
+PKGSEC=net
+PKGDEP="v2ray qt-5 openssl protobuf gprc"
+BUILDDEP="devel-base cmake git pkg-config"
+PKGDES="Cross-platform V2ray Client written in Qt (Stable Release)"
```
Comment (MingcongBai) [inline extra-network/qv2ray/autobuild/defines]: This `PKGDES` is inaccurate, dropping "(Stable Release)" will do.

File: extra-network/qv2ray/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,5 @@
+PKGNAME=Qv2ray
+PKGSEC=net
+PKGDEP="v2ray qt-5 openssl protobuf gprc"
```
Comment (MingcongBai) [inline extra-network/qv2ray/autobuild/defines]: Please double check if dependency is correct and complete - as we have discussed before.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-network/qv2ray/autobuild/defines (hunk 2 above)
Comment (PegionFish) [reply extra-network/qv2ray/autobuild/defines]: Working on it

File: extra-network/qv2ray/autobuild/defines (hunk 1 above)
Comment (PegionFish) [reply extra-network/qv2ray/autobuild/defines]: Fixed

File: extra-network/qv2ray/autobuild/defines (hunk 2 above)
Comment (PegionFish) [reply extra-network/qv2ray/autobuild/defines]: Double checked.

**Pull request closed without merging**